include/libopencm3/stm32/l0/rcc.h

```diff
@@ -505,32 +507,40 @@ enum rcc_periph_clken {
 	RCC_TIM21 = _REG_BIT(0x34, 2),
 	RCC_TIM22 = _REG_BIT(0x34, 5),
 	RCC_MIFI = _REG_BIT(0x34, 7),
+	RCC_FW = _REG_BIT(0x34, 7),
 	RCC_ADC1 = _REG_BIT(0x34, 9),
+	RCC_ADC = _REG_BIT(0x34, 9),
```

Comment:
this was explicitly left out to discourage the adc/adc1 division making it harder to have code run on multiple targets.  The intent was that everyone should just use adc1 even if they only had one adc.